src/com/superiornetworks/icarus/commands/Command_rawsay.java

```diff
@@ -0,0 +1,45 @@
+package com.superiornetworks.icarus.commands;
+
+import com.superiornetworks.icarus.ICM_Rank;
+import com.superiornetworks.icarus.ICM_Utils;
+import net.pravian.bukkitlib.command.BukkitCommand;
+import org.apache.commons.lang.StringUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandParameters(name="rawsay",description="Say a raw message in chat.",usage="/rawsay <message>",rank=ICM_Rank.Rank.SENIOR)
+public class Command_rawsay extends BukkitCommand
+{
+    @Override
+    public boolean run(CommandSender sender, Command cmnd, String string, String[] args)   
+   {
+       Player p = (Player) sender; 
```

Comment:
You need to use instanceof player before casting sender to player. This would throw a NPE if it was sent from console, I believe. You would need to isRankOrHigher thing in an instanceof check as well.